Gopkg.toml

```diff
@@ -27,12 +27,16 @@
 
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
   name = "github.com/giantswarm/apiextensions"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
+
+[[override]]
```

Comment:
This override pushes dep to resolve matching version of the transitive k8s.io/api dependency - otherwise it chooses master.